app/models/itt_subject_set.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class IttSubjectSet
```

Comment:
Suggestion: Should this go in the `LevellingUpPremiumPayments` namespace if it is only concerned with the LUP claim and no others, or the `EarlyCareerPayments` namespace alongide the `SlugSequence`?

Also it seems to be more of a view helper, since it's not used for any business logic e.g. validation or eligibility calculation. Or should we be validating the inputs against this set as well in the controller?

Reply:
Annoyingly it's not unique to either namespace. It's *mostly* about LUP but where you see foreign languages make a reappearance one year that's because of ECP.

Reply:
I don't think it needs a namespace, but it does make sense to me to be put in a helper.

Reply:
Also, you are right in that a controller somewhere should be validating that the subject is valid for a particular year but I've not done that now (I was going to kick that down the road) but I can work on that here if you like?

Reply:
If this is intended to be used in follow up PRs then I don't mind too much.